internal/store/pod.go

```diff
@@ -1125,14 +1126,34 @@ func createPodInitContainerStatusWaitingReasonFamilyGenerator() generator.Family
 	)
 }
 
+func createPodAnnotationsGenerator(allowAnnotations []string) generator.FamilyGenerator {
+	return *generator.NewFamilyGenerator(
+		"kube_pod_annotations",
+		"Kubernetes annotations converted to Prometheus labels.",
```

Comment:
nit for consistency, these should be defined in variables.

Reply:
I used the pattern by of the other functions of the file.

Reply:
Still, would you mind changing it so that we have a more consistent codebase? Labels metrics of this same file should also be put in variables.

Reply:
I feel this is something to do in another PR. This one is already huge enough.